package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@codeandpepper/janush",
-  "version": "1.0.6",
+  "version": "1.0.7",
```

Please, change it in order to avoid conflicts with my changes
```suggestion
  "version": "1.0.8",
```